lambdas/reset-identity/src/main/java/uk/gov/di/ipv/core/resetidentity/ResetIdentityHandler.java

```diff
@@ -112,7 +114,16 @@ public Map<String, Object> handleRequest(ProcessRequest event, Context context)
 
             // Make sure we do this before deleting the credentials!
             String userName = null;
-            if (isUserInitiated) {
+            CriResponseItem f2fRequest = criResponseService.getFaceToFaceRequest(userId);
+
+            if (isUserInitiated && f2fRequest != null) {
+                VcStoreItem credential =
+                        verifiableCredentialService.getVcStoreItem(
+                                clientOAuthSessionItem.getUserId(), FRAUD_CRI);
```

Comment:
Why limit this to the fraud VC for F2F?

Reply:
I've chatted to Joe about this as using the Fraud VC seems fragile to future journey map changes.
Could you please instead add an overload of `UserIdentityService.findIdentityClaim()` that takes a flag to indicate whether to only use evidenceVCs or all VCs (and default the flag to the current behaviour).
Then you can update the old `ResetIdentityHandler.getUserName()` to `ResetIdentityHandler.getUnconfirmedUserName()` and use that for both emails.